### DISCUSSION TITLE
Status Badge not working on mobile app

### DISCUSSION THREAD

**Discussion by kolappannathan** [community/community#31619] [Mobile] [answered]
GitHub Actions status badges for private repos is not working on mobile app.

![Screenshot_20220904-090810_GitHub.jpg](https://user-images.githubusercontent.com/8157105/188296261-5d05258a-0409-4fcc-b05b-1b7c7e679a2a.jpg)

**Comment by taki-on** [comment]
Hi! Thank you for the report. Would you mind giving us a bit more details about the issue? I believe it is about the pull request checks you can find in a pull request checks section and its full list. Do you mean that those status icons are wrong, or that they do not update live? 

I am looking forward to hearing you back!

**Reply by kolappannathan** [reply]
Hi,

I am referring to the status badge of a GitHub action that I have embedded into the Readme file of my project.

The badge was created from the GitHub actions page as shown below

![image](https://user-images.githubusercontent.com/8157105/188175321-5fa6c5cc-fffa-4c6a-8354-63a330058f1d.png)

I updated my post to improve clarification

**Reply by OMikkel** [reply]
On mobile the markdown does not style properly
![image](https://user-images.githubusercontent.com/57924435/188251935-bb0bd750-5164-470f-8101-9850fae4ef5f.jpeg)

**Reply by kolappannathan** [reply]
For me the image itself is not displayed.

![Screenshot_20220904-090810_GitHub.jpg](https://user-images.githubusercontent.com/8157105/188296142-915faf0e-71be-42f0-a524-40cc62fcb227.jpg)

**Comment by taki-on** [comment, marked as answer]
Thank you very much for the details! I will file this as a bug and triage it with the team.

**Reply by jsoref** [reply]
Please select this as the answer.

Fwiw, I filed a related ticket:
https://github.com/orgs/community/discussions/44627

**Comment by zxul767** [comment]
I thought this was a general issue in mobile apps for any repo, but I have just confirmed it happens only in private repos.

**Comment by joshmurrayeu** [comment]
Hi all, just wondering if there has been any advancements with this report? Thanks!

CC @taki-on

**Reply by nathan-brainkey** [reply]
I am also having this issue still. Thought I'd see if there's been any movement on it.

**Comment by bxbdbdbj** [comment]
ً

**Comment by cmcnulty** [comment]
I'm also seeing this issue in my private repo, would love to see an update

**Reply by mark-trigg** [reply]
Still an issue 2 years later...

**Comment by andrian-vangelov** [comment]
> Please select this as the answer.
> 
> Fwiw, I filed a related ticket: https://github.com/orgs/community/discussions/44627




The problem still persists.

It has been more than two years now, do you have any news on that?
